cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -2354,6 +2351,32 @@ pub(super) mod test {
         }
     }
 
+    fn has_null_fields(json_str: &str) -> bool {
+        type Value = serde_json::Value;
+        let json_value: Value = serde_json::from_str(json_str).unwrap();
+
+        // Recursively check if any field in the JSON is `null`
+        fn check_for_null(value: &Value) -> bool {
```

Comment:
Just FYI you can do a `use Value::*` and then only need to match on the variant names.